### DISCUSSION TITLE
時代發展和綠建築

### DISCUSSION THREAD

**Discussion by Lipunpun** [1101-GS4719-C/Python-Community-GS4719-C#1175] [閒聊八卦]
隨著時代的發展，為了節能減碳，現在的人們都知道綠建築要做，但不知道為什麼要這樣做，也不知道這樣做背後帶來什麼價值，我就自然環境、社會環境以及生活空間，三點進行分析：

第一個，自然環境的問題，已經不是所謂的氣候變遷，而是非常極端的氣候了，人不能離開建築存活。

社會環境，能源，核能好嗎，十年前核能是救星，現在慢慢被遺棄，太陽能很好，但另一派說製造過程也不是會產生毒素嗎，後續回收時造成污染，不單純已經是污染、能源的問題更重要的是要如何用更宏觀的角度來探討永續到底是什麼。

空間，對環境的責任，現在很少人在談環保了，過去可能會覺得地球只有一個，所以我們要如何愛護他、保護他，但換個角度想，我們個人不也是一個嗎，所以在這個過程中，環境保護已經逐漸被永續發展所取代了，人類文明在發展的過程中要如何維持自己的生活品質保護自己的文化，因此聯合國推動了永續發展目標，其中地11項永續城市以及社區，3 6 7 9 11 12 13 15這些和其他指標也有間接的關係。

雖然身為一個大學生，還不能為環境做什麼改變，但我認為具備這種永續發展的基本素養還是很好的😆

**Comment by Lipunpun** [comment]
綠建築真的是現在許多賣房廣告的噱頭，出發點很好，但我覺得如果可以就不同環境，從中作設計上的變化，會從盲目的一直用同一種工法下去做，對環境才是好的發展。

**Comment by CCello** [comment]
非常贊同您對永續發展的看法和認識，建築行業作為人類最主要的能源消費和溫室氣體排放行業之一，對環境的影響非常大。因此，推動綠色建築是實現永續發展的重要措施之一。

在自然環境方面，綠色建築能夠最大限度地減少能源消耗和溫室氣體排放，減緩全球氣候變化，同時還能減少對自然資源的消耗，保護生態環境。

在社會環境方面，綠色建築能夠改善室內空氣質量，提高建築物的使用價值和舒適度，提高居民的生活質量，同時還能創造更多的就業機會，推動綠色經濟發展。

在生活空間方面，綠色建築可以提供更優質的居住環境和舒適度，更好地滿足人們的生活需求和期望，同時還能創造更多的社會文化價值，促進人類的健康和幸福感。

總之，綠色建築是推動永續發展和實現可持續性的重要手段之一，可以從多個方面為我們的自然環境、社會環境和生活空間帶來價值和影響。而作為未來的建築從業者和社會公民，我們應該不斷學習、探索和實踐，為推動綠色建築和永續發展做出自己的貢獻

**Comment by KevinCheng42** [comment]
從2015年起，聯合國便開始推廣永續發展目標，讓大眾重新審視綠建築的必要性。我認為若要讓住宅完全以綠建築的標準打造有其困難，包括成本高以及工法複雜造成工時長等問題，然而，我們可以實踐綠建築指標中的幾個項目，如住宅周邊綠化以及利用回收再利用的建材建造等，追求永續發展～

**Comment by YoYooHuang** [comment]
自然環境和能源都正在被人類磨耗，如果我們不趕快做出改變，人類遲早會滅絕。我覺得最簡單的就是，只有每個人都盡一己之力，對環境保護做出一點貢獻，才能真正的解決這個問題，而不是只有推動這個推動那個，要行動才會有所改變！
